air_to_breath_resources/utils/sensor.py

```diff
@@ -82,70 +85,66 @@ def simulate(self, low_value=None, high_value=None):
 
 
 class SocketSensor:
-    LOW_BOUND = 0
-    HIGH_BOUND = 5
+    CONFIG_PATH = Path(os.getenv('PYTHONPATH')) / 'air_to_breath' / 'configs' / 'config.json'
 
-    def __init__(self, host, port):
-        self.host = host
-        self.port = port
+    ERROR_TEMPLATE = NotImplemented
+    VALUE_TEMPLATE = NotImplemented
 
-        self.low_bound = self.LOW_BOUND
-        self.high_bound = self.HIGH_BOUND
+    def name(self):
+        return NotImplemented
 
+    def __init__(self):
         self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
 
-    def set_value(self, sent_value: int):
+    @property
+    def _config_json(self):
+        with open(self.CONFIG_PATH) as f:
+            return json.load(f)
+
+    @property
+    def low_bound(self):
+        return self._config_json['threshold'][self.name]['min']
+
+    @property
+    def high_bound(self):
+        return self._config_json['threshold'][self.name]['max']

```

Comment:
all properties can be cached

Reply:
In later tests, if we want to change the bounds we will just need to change the config.

Reply:
Ok the config still can be cached

Reply:
but we will change the file, since alot of places access it (rotest and this)

Reply:
You can change it whenever you want.
But from the moment you start the test and until it over, you will not change it.
The cache happen only for current run

Reply:
I want to change it dynamiclly.
In the middle of the test i want to write to the file values and thus change the high and low values.

Reply:
This doesn't make any sense... this is not the purpose of config file

Reply:
I know, but i rather have one source that unites all the data instead of duplicate it from a couple of sources.
I abuse the config file a little bit, but i think its worth it.

Reply:
You can use the config as default value, and take it from different place when you want to change it. There is a limit to any trick.